benchmark/esm/import-meta.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const path = require('path');
+const common = require('../common');
+const assert = require('assert');
+const bench = common.createBenchmark(main, {
+  n: [1000],
+});
+
+const file = path.resolve(__filename, '../../fixtures/esm-dir-file.mjs');
+async function load(array, n) {
+  for (let i = 0; i < n; i++) {
+    array[i] = await import(`${file}?i=${i}`);
+  }
```

Comment:
A comment here indicating why we're awaiting each individual import as opposed to something like the following would be helpful...

```
for (let i = 0; i < n; i++) {
  array[i] = import(`...`);
}
await Promise.all(array);
```

Reply:
I don’t actually know why, the original benchmarks were already like this, I guess they didn’t want promise performance/races dominate the benchmark?

Reply:
Maybe @jsumners or any of the reviewers of https://github.com/nodejs/node/pull/48740 know why the benchmark was waiting for individual promises (actually I see that you are also one of the reviewers :)) my experience with the startup benchmark though is that if you add concurrency the benchmark can be quite flaky.

Reply:
heh, yeah I remember reviewing it originally. Consider this non-blocking.

Reply:
I just don't `Promise.all` unless there is a specific reason to.